pkg/osutil/osutil.go

```diff
@@ -73,19 +73,43 @@ func EnsureFile(file string) error {
 }
 
 type TemplateWriter interface {
```

Comment:
Maybe we should throw the template writer Interface and implementations in it's own go file

Reply:
moved templatewriter to its own directory and created a writers directory to house the imeplemetations to not create cyclical imports